static/src/javascripts/projects/commercial/modules/close-disabled-slots.js

```diff
@@ -21,25 +21,43 @@ const closeDisabledSlots = once((): Promise<void> => {
 });
 
 const mpuCandidateClass: string = 'fc-slice__item--mpu-candidate';
-const mpuHiderClass: string = 'fc-slice__item--no-mpu';
 const mpuCandidateSelector: string = `.${mpuCandidateClass}`;
 
-const shouldDisableAdSlotWhenAdFree = adSlot =>
+const shouldRemoveAdSlotWhenAdFree = () => commercialFeatures.adFree;
+
+const shouldRemoveMpuWhenAdFree = mpuCandidate =>
     commercialFeatures.adFree &&
-    (adSlot.className.toLowerCase().includes(mpuCandidateClass) ||
-        !adSlot.className.toLowerCase().includes('merchandising'));
+    mpuCandidate.className.toLowerCase().includes(mpuCandidateClass);
+
+const shouldRemoveFaciaContainerWhenAdFree = faciaContainer => {
+    const att = faciaContainer.getAttribute('data-component');
+    return (
+        commercialFeatures.adFree &&
+        att &&
+        att.indexOf('commercial-container') > -1
```

Comment:
is there a clearer/ non magic number way of testing that 'att' contains 'commercial-container' ?

Reply:
https://stackoverflow.com/a/1789952/2118700